core/dbt/events/functions.py

```diff
@@ -171,6 +172,22 @@ def create_text_log_line(e: T_Event, msg_fn: Callable[[T_Event], str]) -> str:
     return log_line
 
 
+def create_file_text_log_line(e: T_Event, msg_fn: Callable[[T_Event], str]) -> str:
+    log_line: str = ''
+    # Create a separator if this is the beginning of an invocation
+    if type(e) == MainReportVersion:
+        separator = 30 * '='
+        log_line = f'\n\n{separator} {e.get_ts()} | {get_invocation_id()} {separator}\n'
```

Comment:
yes!! with the `invocation_id` too!! I really like this

@nathaniel-may may have thoughts about implementation. Given that `MainReportVersion` really truly is a one-of-a-kind event, and ought to be the very first one fired, this approach makes some sense to me

Reply:
yeah if it's truly one of a kind and only happens in this text-file combo, this approach looks like the right way to do it.